plugins/mentions.md

```diff
@@ -93,6 +91,59 @@ tinymce.init({
 });
 ```
 
+### `mentions_item_type`
+
+{{site.requires_5_6v}}
+
+This option allows you to specify what kind of user interface item to use when displaying the list of users. The `name` item will just display the user's name, while the `profile` item can also display an optional image and description. Depending on what this option is set to, additional properties may be required for the user object provided via [mentions_fetch](#mentions_fetch). Refer to the [User properties](#userproperties) table for more details.
```

Comment:
We shouldn't be referring to "you" or anything like that in technical documentation. I know there's a lot of older parts that still do that, but we should avoid it in newer docs.
```suggestion
This option controls what kind of user interface item to use when displaying the list of users. The `name` item will just display the user's name, while the `profile` item can also display an optional image and description. Depending on what this option is set to, additional properties may be required for the user object provided via [mentions_fetch](#mentions_fetch). Refer to the [User properties](#userproperties) table for more details.
```

Reply:
@tylerkelly13 will probably have a better suggestion than `controls` here 😛